common/main/cpp/native_c4listener.cc

```diff
@@ -177,6 +179,7 @@ static bool doDecryptCallback(
     int n = (int) input.size;
 
     jbyteArray encData = env->NewByteArray(n);
+    if (encData == nullptr) return false;
```

Comment:
If this ever happens you are probably never going to figure out why without some kind of message in the logs

Reply:
Yeah.... that's a good point.  Will add in a future commit